gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/ImmunizationChecker.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+import uk.nhs.adaptors.connector.dao.ImmunizationSnomedCTDao;
+
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Component
+public class ImmunizationChecker implements ImmunizationCheckerInterface {
```

Comment:
I'd work on the names:
I'd call the interface: ImmunisationChecker and it's implementation - DatabaseImmunisationChecker.

If we call the implementation ImmunisationChecker then it doesn't really say how it's being implemented and doesn't leave too much space for other implementations